libtiledbsoma/src/soma/soma_multiscale_image.cc

```diff
@@ -24,11 +24,30 @@ using namespace tiledb;
 void SOMAMultiscaleImage::create(
     std::string_view uri,
     std::shared_ptr<SOMAContext> ctx,
+    const SOMACoordinateSpace& coordinate_space,
     std::optional<TimestampRange> timestamp) {
     try {
         std::filesystem::path image_uri(uri);
-        SOMAGroup::create(
+        auto group = SOMAGroup::create(
             ctx, image_uri.string(), "SOMAMultiscaleImage", timestamp);
+
+        // Set spatial encoding metadata.
+        group->set_metadata(
+            SPATIAL_ENCODING_VERSION_KEY,
+            TILEDB_STRING_UTF8,
+            static_cast<uint32_t>(SPATIAL_ENCODING_VERSION_VAL.size()),
+            SPATIAL_ENCODING_VERSION_VAL.c_str(),
+            true);
+
+        // Set coordinate space metadata.
```

Comment:
```suggestion
        // Set coordinat-space metadata.
```

Reply:
I don't think coordinate space should be hyphenated.